include/vulkan/vk_icd.h

```diff
@@ -111,6 +111,12 @@ typedef struct {
 } VkIcdSurfaceXlib;
 #endif // VK_USE_PLATFORM_XLIB_KHR
 
+#ifdef VK_USE_PLATFORM_ANDROID_KHR
```

Comment:
As this file resides in LVL repo, better to make this change in LVL instead of VT.